### PR TITLE
2045 content state editorial

### PR DESCRIPTION
**Pull request by azaroth42**
Editorial changes for #2045 from Rob.

Notably: lots of minor typos, clarifications. Restructure to format, encoding, protocol, processing.

### CODE REVIEW

**Review by mixterj — DISMISSED** [review]

**Comment by azaroth42** [comment]
Preview: https://preview.iiif.io/api/2045_content_state_editorial/api/content-state/0.9/

**Review by kirschbombe — APPROVED** [review]

**Review by tomcrane — APPROVED** [review]
I did have one comment but on further reflection I retracted it...

File: source/api/content-state/0.9/index.md [hunk 1]
```diff
@@ -49,25 +49,29 @@ This is a work in progress and may change without notice. Implementers should be
 ## 1. Introduction
 {: #introduction}
 
-This specification provides a way of describing a [IIIF Presentation API][prezi-api] resource, or a part of a resource, in a compact format that can be used to initialize the view of that resource in any client that implements this specification. This description is the _content state_. This specification provides the format of the _content state_, and mechanisms for passing it between applications regardless of their different user interfaces and capabilities.
+This specification provides a method to reference a [IIIF Presentation API][prezi-api] resource, or a part of a resource, in a compact format that can be used to initialize the view of that resource in any client. This description is called the _content state_. This specification provides the format of the _content state_, and mechanisms for passing it between applications regardless of their different user interfaces and capabilities.
```
Comment (zimeon) [inline source/api/content-state/0.9/index.md:52]: Would it be clearer to say "[way|method] to refer to a"?

File: source/api/content-state/0.9/index.md [hunk 2]
```diff
@@ -230,13 +224,13 @@ For these reasons, this specification defines a two-step encoding that uses both
...
+The process to encode a content state is:
 
 * encode to a UTF-8 string described by [encodeURI][org-ecma-encodeuri]
 * encode the resulting UTF-8 string as [base64url][org-rfc-4648-5]
 * remove any "=" padding characters from the end
 
-To decode:
+Conversely to decode a content state:
 
 * restore any missing "=" padding characters to the end of the string
```
Comment (zimeon) [inline source/api/content-state/0.9/index.md:235]: Should this be specific about how to tell what "=" characters need be added? The "missing" ones seems vague. Perhaps:
* restore any removed "=" padding characters to the end of the string to make it a multiple of 4 characters long

**Review by zimeon — COMMENTED** [review]
I approve overall but have a couple of comments/suggestions that may be better dealt with separately. This draft also needs to have ALL internal section references linked

**Review by zimeon — APPROVED** [review]

**Review by jpstroop — APPROVED** [review]

**Review by mcwhitaker — APPROVED** [review]

**Review by mikeapp — APPROVED** [review]

**Pull request merged**